CLAUDE.md

```diff
@@ -58,22 +58,16 @@ make push-all-images
 
 ### Helm Deployment
 
-The system supports three deployment modes:
+The system supports two deployment modes:
 
 ```bash
-# 1. Development Mode (Direct HTTP)
-make helm-install-dev NAMESPACE=your-namespace \
-  LLM=llama-3-2-1b-instruct \
-  SLACK_SIGNING_SECRET="your-secret" \
-  SNOW_API_KEY="your-key"
-
-# 2. Testing Mode (Mock Eventing)
+# 1. Testing/Development Mode (Mock Eventing - Default)
 make helm-install-test NAMESPACE=your-namespace \
```

Comment:
should we change the name here to `helm-install` if we expect it to be the default and we no longer have a dev mode?